compiler/tests/org/jetbrains/kotlin/generators/tests/GenerateRangesCodegenTestData.java

```diff
@@ -119,7 +119,7 @@ private static String renderTemplate(String template, int number, String element
                 .replace("\n", LineSeparator.getSystemLineSeparator().getSeparatorString());
     }
 
-    private static final List<String> WHITELISTED_FOR_JVM_IR_BACKEND = Collections.singletonList("overflowZeroDownToMaxValue.kt");
```

Comment:
Inlining mangled external declarations is currently broken.

Inlining external declarations uses descriptors to look up the underlying bytecode for inlining. Since I'm doing name mangling for inline class functions at the IR level, these descriptors are replaced by WrappedDescriptors which the inliner currently does not handle.

The fact that this particular test was working before was a bit of a happy coincidence - it didn't really depend on the inline class machinery at all.

I'm planning to submit a separate PR for this later, since the inliner seems to be very much a work in progress.